### ISSUE TITLE
Scan for protection: Path could not be scanned!

### ISSUE THREAD

**Issue by MichaelDavidHarry** [SabreTools/DICUI#169]
I get the 'Path could not be scanned!' message every time I try to use the Scan for protection feature.

Here is what the log output says:
```Populating systems, 113 systems found.
Changed system to: Unknown
Scanning for drives..
Trying to detect media type for drive I.. detected CD-ROM.
Supported media speeds: 1,2,3,4,6,8,12,16,20,24,32,40,44,48,52,56,72
Setting drive speed to: 20
Trying to detect system for drive I.. unable to detect.
Found 4 drives: G, H, I, J
Scanning for copy protection in I
Detected the following protections in I:

Path could not be scanned!
```

My drive is a PLEXTOR CD-R PX-W4824A over a USB adaptor. I get this same error when I try with any of my other drives as well. I'm trying to test this with a copy of the original The Sims which should have some kind of SafeDisc on it.

I did some fiddling around with the DICUI project in Visual Studio and the exception seems to be coming from the ProtectionFind.Scan(path); code on line 925 of Validators.cs.

I had some issues building the project from source, and I had to bring in the BurnOutSharp, libmspack4n, and LessIO libraries as projects to get this to build. I'm using Visual Studio Community 2019. Any tips on getting it to build from what is in the git repository would be appreciated.

Thanks for this program!

**Comment by mnadareski** [comment]
This has already been fixed in some of the recent BurnOutSharp and DICUI builds. Please try them out and report back if it's still failing.

**Comment by mnadareski** [comment]
If you want it to build, you need to Nuget restore and it will bring in the required stuff. Otherwise, just use the AppVeyor builds linked on the main page. Linked here for convenience: https://ci.appveyor.com/project/mnadareski/dicui/build/artifacts

**Comment by MichaelDavidHarry** [comment]
Thanks! This build (https://ci.appveyor.com/project/mnadareski/dicui/builds/28395164/artifacts) seems to work OK for me.

I think my build issues were caused by some issues with COM objects that couldn't be found, I think this got resolved when I installed the Windows Driver Kit. (At least I think that fixed it, but I'm not totally sure. I installed a couple of other things as well.)

Thanks again!
